PSix/FacebookEventParser.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  FacebookEventParser.swift
+//  PSix
+//
+//  Created by Turzion, Avihu on 4/20/15.
+//  Copyright (c) 2015 PayPal. All rights reserved.
+//
+
+import Foundation
+
+class FacebookEventsParser {
```

Comment:
If the reasoning behind making it a `struct` is to have static methods then it doesn't matter now because Swift 1.2 allows for static methods in classes as well.

My thinking behind this was to make it like the [`StringTokenizer`](https://docs.oracle.com/javase/7/docs/api/java/util/StringTokenizer.html) in Java where you feed the `string` and it becomes an immutable representation of the processed data. Having it as a static function is just as valid. If style-wise you think it's more appropriate, I will happily change it :)